docs/source/commands/dev.mdx

```diff
@@ -187,12 +187,25 @@ By default, the router's health check endpoint is disabled in `rover dev`. You c
 
 To configure advanced router functionality like CORS settings or header passthrough for subgraphs, you can pass a valid [router configuration YAML file](/router/configuration/overview#yaml-config-file) to `rover dev` via the `--router-config <ROUTER_CONFIG_PATH>` argument.
 
-### Enterprise features
+### GraphOS Router features
```

Comment:
Since some of these features are now also available on free plans updating this. (See https://www.apollographql.com/docs/graphos/routing/graphos-features)